### PR TITLE
dellctl schedule commands

### PR DESCRIPTION
**Pull request by bpjain2004**
# Description
Add new dellctl commands to create and manage schedules for backups.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|dell/csm#491 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/references/cli/_index.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ This document outlines all dellctl commands, their intended use, options that ca
 | [dellctl restore create](#dellctl-restore-create) | Restore an application backup |
 | [dellctl restore delete](#dellctl-restore-delete) | Delete application restores |
 | [dellctl restore get](#dellctl-restore-get) | Get application restores |
+| [dellctl schedule](#dellctl-schedule) | Allows to manipulate schedules |
```
Comment (rsedlock1958) [inline content/docs/references/cli/_index.md:26]: Add "you" after "Allows"

File: content/docs/references/cli/_index.md [hunk 2]
```diff
@@ -532,3 +537,204 @@ Get restores with their names
...
+
+Delete one or more schedules
+
+##### Flags
+
+```
+      --all                 Delete all schedules
+      --cluster-id string   Id of the cluster managed by dellctl
+      --confirm             Confirm deletion
+  -h, --help                help for delete
```
Comment (rsedlock1958) [inline content/docs/references/cli/_index.md:675]: Change "help" to Help"

File: content/docs/references/cli/_index.md [hunk 3]
```diff
@@ -532,3 +537,204 @@ Get restores with their names
...
+
+### dellctl schedule get
+
+Get schedules
+
+##### Flags
+
+```
+      --cluster-id string   Id of the cluster managed by dellctl
+  -h, --help                help for get
```
Comment (rsedlock1958) [inline content/docs/references/cli/_index.md:719]: Change "help" to "Help"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There are several minor edits required.

File: content/docs/references/cli/_index.md (hunk 1 above)
Comment (bpjain2004) [reply content/docs/references/cli/_index.md]: > Add "you" after "Allows"

done

File: content/docs/references/cli/_index.md (hunk 2 above)
Comment (bpjain2004) [reply content/docs/references/cli/_index.md]: done

File: content/docs/references/cli/_index.md (hunk 3 above)
Comment (bpjain2004) [reply content/docs/references/cli/_index.md]: updated

**Review by rsedlock1958 — DISMISSED** [review]

**Review by sharmilarama — DISMISSED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Review by prablr79 — APPROVED** [review]
lgtm

**Pull request merged**